samples/azure-quantum/check-ghz/CheckGHZ.ipynb

```diff
@@ -0,0 +1,191 @@
+        "   if not (r0 == r1 and r1 == r2)\n",
+        "```\n",
+        "2. **Classical comput**e: here an arithmetic operation is performed\n",
+        "```Q#\n",
+        "   set mismatch += 1;\n",
+        "```\n",
+        "3. **While the physical qubits are still coherent**, #1 and #2 are performed \n",
+        "\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "%azure.connect \"/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.Quantum/Workspaces/workspaceName\" location=\"westus\""
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## Define the CheckGHZ operation\n",
+        "Please note that you will see a warning displayed (QS5026). It can be ignored. It will be addressed in a subsequent release of the Quantum Development Kit."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "microsoft": {},
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "open Microsoft.Quantum.Measurement;\n",
+        "open Microsoft.Quantum.Arrays;\n",
+        "open Microsoft.Quantum.Convert;\n",
+        "\n",
+        "operation CheckGHZ() : Int {\n",
+        "    use q = Qubit[3];\n",
+        "    mutable mismatch = 0;\n",
+        "    for _ in 1..10 {\n",
+        "        H(q[0]);\n",
+        "        CNOT(q[0], q[1]);\n",
+        "        CNOT(q[1], q[2]);\n",
+        "\n",
+        "        // Measures and resets the 3 qubits\n",
+        "        let (r0, r1, r2) = (MResetZ(q[0]), MResetZ(q[1]), MResetZ(q[2]));\n",
+        "\n",
+        "        // Adjusts classical value based on measurement results\n",
+        "        if not (r0 == r1 and r1 == r2) {\n",
+        "            set mismatch += 1;\n",
+        "        }\n",
+        "    }\n",
+        "    return mismatch;\n",
+        "}"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Submit the operation\n",
+        "For 50 shots, as configured below, the following will use about 10.65 units (HQCs or eHQCs).<br/>\n",
+        "If the job is not completed after the timeout (300 seconds), you can query the status followed by querying the output.<br/>\n",
```

Comment:
Took me a minute to understand the phrase "query the status followed by querying the output" - maybe say "you can query the status of the job and read the output once it completes"

Reply:
Link is not live yet